enterprise/admin/views.py

```diff
@@ -203,6 +219,50 @@ def _handle_bulk_upload(cls, enterprise_customer, manage_learners_form, request)
                     list_of_emails=", ".join(duplicate_emails)
                 )
             )
+        return list(emails) + already_linked_emails
+
+    @classmethod
+    def _enroll_users(cls, emails, course_details, mode, request):
+        """
+        Enroll the users with the given email addresses to the course specified by course_details.
+        """
+        enrolled = []
+        non_existing = []
+        errors = []
+        for email in emails:
+            try:
+                username = User.objects.get(email=email).username
+            except User.DoesNotExist:
+                non_existing.append(email)
+                continue
+            try:
+                enroll_user_in_course(username, course_details, mode)
+            except HttpClientError as exc:
+                errors.append(email)
+                error_message = json.loads(exc.content.decode()).get("message", "No error message provided.")
+                logging.error(
+                    "Error while enrolling user %(user)s: %(message)s",
+                    dict(user=username, message=error_message),
+                )
+            else:
+                enrolled.append(email)
+        count = len(enrolled)
+        course_id = course_details["course_id"]
+        messages.success(request, ungettext(
```

Comment:
@haikuginger This was intentional, and it is in line with what the bulk linking function does, but I don't care either way.  Since you cared enough to comment here, I've changed it now. :)